lib/assertions.js

```diff
@@ -476,6 +495,25 @@ module.exports = function (expect) {
         if (value === '') {
             throw new Error("The '" + expect.testDescription + "' assertion does not support a suffix of the empty string");
         }
+        var isTruncated;
```

Comment:
Which bit did you want factored into a utils function? Agreed that'd make the assertions much clearer to read - my only thought was I think there were two separate algorithms for checking truncations from the beginning or end of the string. Would it be a flag to the factored out fn?